projects/ngx-form-object/src/lib/form-store/form-store.ts

```diff
@@ -6,28 +6,29 @@ export class FormStore extends FormGroup {
 	private _formObject: FormObject;
 	private _isSubmitted: boolean = false;
 
-	get isChanged(): boolean {
+	public get isChanged(): boolean {
 		return this.attributesDidChange || this.belongsToPropertiesDidChange || this.hasManyPropertiesDidChange;
 	}
 
-	set formObject(formObject: FormObject) {
+	public set formObject(formObject: FormObject) {
 		this._formObject = formObject;
 	}
 
-	get formObject(): FormObject {
+	public get formObject(): FormObject {
 		return this._formObject;
 	}
 
-	get model(): any {
+	public get model(): any {
 		return this.formObject.model;
 	}
 
-	get isSubmitted(): boolean {
+	public get isSubmitted(): boolean {
 		return this._isSubmitted;
 	}
 
 	public save(): Observable<any> {
 		this._isSubmitted = true;
+		// eslint-disable-next-line no-underscore-dangle
```

Comment:
I think this rule can be removed. As this is an open source library, it is useful to hide some thing behind `_` to avoid potential clashes.